src/CampaignsAdmin/CampaignDetails.tsx

```diff
@@ -104,6 +98,26 @@ export function CampaignDetails() {
     }
   };
 
+  /**
+   * Removes photo file on supabase and updates locationInfo JSON
+   */
+  const removePhotoOnSupabase = async (locationID: number, imageURL: string) => {
```

Comment:
Gotta make sure that you check on permissions before making this public. Currently as-written, any user can delete all photos and db rows using the supabase client.